atd-events/crash_update_jurisdiction/app.py

```diff
@@ -67,6 +69,46 @@ def get_jurisdiction_flag(data: dict) -> str:
         return "N"
 
 
+def get_city_id(data: dict) -> Optional[int]:
+    """
+    Returns the city id of the record in question
+    :param dict data: The record in question
+    :return Optional[int]: The city id as an integer, 0 if it can't find it.
+    """
+    try:
+        return data["event"]["data"]["new"]["city_id"]
+    except (TypeError, KeyError):
+        return None
+
+
+def get_original_city_id(data: dict) -> Optional[int]:
+    """
+    Returns the original city id of the record in question
+    :param dict data: The record in question
+    :return Optional[int]: The city id as an integer, 0 if it can't find it.
+    """
+    try:
+        return data["event"]["data"]["new"]["original_city_id"]
+    except (TypeError, KeyError):
+        return None
+
+
+def get_jurisdiction_common_members(a: List[int], b: List[int]) -> Set[int]:
+    """
+    Compares two arrays of integers, and returns a set of unique common members
+    :param List[int] a: An array you want to compare against
+    :param List[int] b: The other array you want to compare against
+    :return Set[int]:
+    """
+    a_set = set(a)
+    b_set = set(b)
+
+    if a_set & b_set:
```

Comment:
I didn't realize that you could compare sets like this! Very handy way to find the common elements. 😎

Reply:
Yea, I had not been using sets before and I think I need to make more use of them going forward. They are a swiss-army knive for data processing